boards/arm/mps2_an521/mps2_an521_nonsecure.dts

```diff
@@ -45,8 +45,8 @@
 		reg = <0x28100000 0x100000>;
 	};
 
-	flash0: flash@100000 {
-		reg = <0x100000 0xDF00000>;
+	flash0: flash@100400 {
```

Comment:
This commit is from #16450 at [here](https://github.com/zephyrproject-rtos/zephyr/pull/16450/commits/f9d10e31e2436c2dc604f2a3fe43d8dd2b67e961) to resolve secure fault while jump into Zephyr.